command_line_provider.go

```diff
@@ -77,7 +77,12 @@ func NewCommandLineProvider(flags *flag.FlagSet, args []string) Provider {
 		m[f.Name] = f.Value.String()
 	})
 
-	return commandLineProvider{Provider: NewStaticProvider(m)}
+	p, err := NewStaticProvider(m)
+	if err != nil {
+		return nil, err
```

Comment:
I can cover this line, but it'll not provide much value, simply a public function returning an error via another public function.